apis/clusters/v1beta1/structs.go

```diff
@@ -66,41 +54,12 @@ type RestoreCustomVPCSettings struct {
 	Network string `json:"network"`
 }
 
-type Options struct {
-	DataNodeSize                 string `json:"dataNodeSize,omitempty"`
-	MasterNodeSize               string `json:"masterNodeSize,omitempty"`
-	OpenSearchDashboardsNodeSize string `json:"openSearchDashboardsNodeSize,omitempty"`
-}
-
-type Cluster struct {
-	// Name [ 3 .. 32 ] characters.
-	Name string `json:"name,omitempty"`
-
-	Version string `json:"version,omitempty"`
-
-	// The PCI compliance standards relate to the security of user data and transactional information.
-	// Can only be applied clusters provisioned on AWS_VPC, running Cassandra, Kafka, Elasticsearch and Redis.
-	PCICompliance bool `json:"pciCompliance,omitempty"`
-
-	PrivateNetworkCluster bool `json:"privateNetworkCluster,omitempty"`
-
-	// Non-production clusters may receive lower priority support and reduced SLAs.
-	// Production tier is not available when using Developer class nodes. See SLA Tier for more information.
-	// Enum: "PRODUCTION" "NON_PRODUCTION".
-	SLATier string `json:"slaTier,omitempty"`
-
-	TwoFactorDelete []*TwoFactorDelete `json:"twoFactorDelete,omitempty"`
-
-	Description string `json:"description,omitempty"`
-}
-
 type ClusterStatus struct {
 	ID                            string                                             `json:"id,omitempty"`
 	State                         string                                             `json:"state,omitempty"`
 	DataCentres                   []*DataCentreStatus                                `json:"dataCentres,omitempty"`
 	CDCID                         string                                             `json:"cdcid,omitempty"`
 	TwoFactorDeleteEnabled        bool                                               `json:"twoFactorDeleteEnabled,omitempty"`
```

Comment:
please correct me if I'm wrong, but I believe these two fields should be removed

Reply:
I believe even entire struct is not being used.